packages/rainbowkit/src/wallets/getWalletConnectConnector.ts

```diff
@@ -0,0 +1,35 @@
+import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
+import { Chain } from '../components/RainbowKitProvider/RainbowKitChainContext';
+
+type SerializedOptions = string;
+const sharedConnectors = new Map<SerializedOptions, WalletConnectConnector>();
+
+type WalletConnectConnectorOptions = ConstructorParameters<
+  typeof WalletConnectConnector
+>[0];
+
+function createConnector(options: WalletConnectConnectorOptions) {
+  const connector = new WalletConnectConnector(options);
+  sharedConnectors.set(JSON.stringify(options), connector);
+  return connector;
+}
+
+export function getWalletConnectConnector({
+  chains,
+  qrcode = false,
+}: {
+  chains: Chain[];
+  qrcode?: boolean;
+}) {
+  const options: WalletConnectConnectorOptions = {
+    chains,
+    options: {
+      qrcode,
```

Comment:
Note that we no longer set the `rpc` option. This is redundant since wagmi handles this internally now via `configureChains`, introduced in v0.4 which is our current peer dep version range. The RPC information is baked into the `chains` option which is also passed in here.

Reply:
I decided I'm going to roll this change back and apply it to https://github.com/rainbow-me/rainbowkit/pull/471 instead since it's related to that work. We can release this PR first as a patch before shipping a minor update.